src/overlays/actors/ovl_En_Viewer/z_en_viewer.h

```diff
@@ -18,7 +18,11 @@ typedef struct {
     /* 0x06 */ u8 actorShadowDrawFuncIndex;
     /* 0x07 */ u8 unk_7;
     /* 0x08 */ u8 drawFuncIndex;
-    /* 0x0C */ FlexSkeletonHeader* unk_C;
+    /* 0x0C */ union {
+                    void* ptr; // prevents compiler warnings
+                    SkeletonHeader* normal;
+                    FlexSkeletonHeader* flex;
+                } unk_C;
```

Comment:
Does the code in this actor use any fields from `SkeletonHeader` or `FlexSkeletonHeader`? If not, we can probably just treat pointers to skeleton headers as `void*` everywhere, and avoid any union or casts.

Reply:
Yep, that works.